jax/random.py

```diff
@@ -226,19 +226,21 @@ def _check_shape(name, shape):
     raise ValueError(msg.format(name, shape))
 
 
-def uniform(key, shape, dtype=onp.float32, minval=0., maxval=1.):
+def uniform(key, shape, dtype=onp.float64, minval=0., maxval=1.):
```

Comment:
nit: can we have a default `shape=()` here

Reply:
Done!